src/components/AlertBanner/__stories__/alertBanner.stories.mdx

```diff
@@ -0,0 +1,208 @@
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import AlertBanner from "../AlertBanner";
+import AlertBannerText from "../AlertBannerText/AlertBannerText";
+import AlertBannerLink from "../AlertBannerLink/AlertBannerLink";
+import AlertBannerButton from "../AlertBannerButton/AlertBannerButton.jsx"
+import { Link } from "../../../storybook/components";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { TOOLTIP, ATTENTION_BOX, TOAST } from "../../../storybook/components/related-components/component-description-map";
+import "./alertBanner.stories.scss";
+
+<Meta
+  title="Components/AlertBanner"
+  component={ AlertBanner }
+/>
+
+<!--- Component template -->
+
+export const alertBannerTemplate = (args) => { 
+  return (<AlertBanner {...args}>
+            <AlertBannerText text={ args.bannerText} />
+            <AlertBannerLink text={ args.linkText } href="https://monday.com" />
+          </AlertBanner>);
+}
+
+<!--- Component documentation -->
+
+# Alert Banner
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Alert banners show pressing and high-signal messages, such as system alerts. They are meant to be noticed and prompt users to take action.
+
+<Canvas>
+  <Story name="Overview"
+         args={{ bannerText: "Alert banner message", linkText: "this is a CTA", className: "monday-storybook-alert-banner_container" }}>
+    { alertBannerTemplate.bind({ }) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ AlertBanner } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Alert banners should have call for action and an option to dismiss.",
+  "Don't include more than one call to action in an alert banner.",
+  "Place the banner on the top of the screen, and push all other content below it.",
+]}/>
+
+<Tip>Alert banners should be reserved only for high-signal, system-level alert messages. For in-app notifications use a <Link href="/?path=/docs/components-toast--overview" size={Link.sizes.SMALL} withoutSpacing>Toast.</Link></Tip>
+
+## Variants
+### Types
+There are 4 types of alert banners: primary, negative, positive, and inverted.
+<Canvas>
+  <Story name="Types">
+    <div className="monday-storybook-alert-banner_column-wrapper monday-storybook-alert-banner_container">
```

Comment:
why do you need here 2 classes? monday-storybook-alert-banner_column-wrapper sounds not related to the variants types?

Reply:
"_column-wrapper" is related to the layout of children inside the container, "_big-container" is responsible for the width of items. We heavily use the different sizes "_big-container" and "_medium-container", and in two places we use containers for AlertBanner, and they have the class "_column-wrapper"